.eslintrc.json

```
@@ -1,5 +1,14 @@
 {
-  "extends": "airbnb-base",
+  "parser": "@typescript-eslint/parser",
+  "parserOptions": {
+    "project": "./tsconfig.json"
+  },
+  "plugins": ["@typescript-eslint"],
+  "extends": [
+    "airbnb-base",
+    "plugin:@typescript-eslint/recommended",
+    "plugin:@typescript-eslint/recommended-requiring-type-checking"
```

```suggestion
    "plugin:@typescript-eslint/all",
    "airbnb-typescript/base"
```

Note that the order does have an effect; I believe the later entries override the earlier entries, which is what I think we want here, but I haven't been able to find any documentation on that and haven't dived into the eslint source code.